components/Alert.css

```diff
@@ -0,0 +1,191 @@
+/* css for nav*/
+:root{
+    --primary-color:#1E40AF;
+    --secondary-color:#BFDBFE;
+}
+body{
+    overflow-x: hidden;
+    overflow-y: hidden;
+    margin: 0;
+}
+#navigation{
```

Comment:
use .class instead of id# in css, because it's creates specificity problem later in site